tasks/plugins.yml

```diff
@@ -83,3 +85,20 @@
       - restart sensu-server service
       - restart sensu-api service
       - restart sensu-enterprise service
+
+  - name: Register available client definitions
```

Comment:
Hey @tculp! 

So sorry about the lag, I just got the TravisCI build working for this repo. It looks like all of the builds for this PR now fail with:
```
TASK [role_under_test : Register available client definitions] *****************
fatal: [localhost -> localhost]: FAILED! => {"changed": false, "cmd": ["ls", "/etc/ansible/roles/role_under_test/tests/data/static/sensu/client_definitions"], "delta": "0:00:00.035682", "end": "2018-01-30 13:02:07.529014", "msg": "non-zero return code", "rc": 2, "start": "2018-01-30 13:02:07.493332", "stderr": "ls: cannot access /etc/ansible/roles/role_under_test/tests/data/static/sensu/client_definitions: No such file or directory", "stderr_lines": ["ls: cannot access /etc/ansible/roles/role_under_test/tests/data/static/sensu/client_definitions: No such file or directory"], "stdout": "", "stdout_lines": []}
```

I believe it's because above on line 22 you used `client-definitions` but here you've used `client_definitions`.